<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ETS2 Autopilot</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="image_front">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>360</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>360</width>
      <height>200</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_autopilot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Autopilot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_ets2_vjoy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Joystick detection</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionUpdater"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <action name="actionSettings">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUpdater">
   <property name="text">
    <string>Updater</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
